README.md

```diff
@@ -1,79 +1,92 @@
-[![charmcraft](https://snapcraft.io/charmcraft/badge.svg)](https://snapcraft.io/charmcraft)
+# Charmcraft
+
+[![Charmcraft](https://snapcraft.io/charmcraft/badge.svg)](https://snapcraft.io/charmcraft)
 [![Tests](https://github.com/canonical/charmcraft/actions/workflows/tests.yaml/badge.svg?event=push)](https://github.com/canonical/charmcraft/actions/workflows/tests.yaml)
 [![Spread](https://github.com/canonical/charmcraft/actions/workflows/spread.yaml/badge.svg?event=push)](https://github.com/canonical/charmcraft/actions/workflows/spread.yaml)
 [![Weekly Spread](https://github.com/canonical/charmcraft/actions/workflows/spread-large.yaml/badge.svg)](https://github.com/canonical/charmcraft/actions/workflows/spread-large.yaml)
 
-# Charmcraft -- easily initialise, pack, and publish your charms
+**Charmcraft** is the command-line tool for initializing, packaging, and publishing
+[Juju charms](https://documentation.ubuntu.com/juju/3.6/reference/charm). With
```

Comment:
I'd rather we don't immediately link to somewhere else; do you think a very brief summary of what a charm is would be better?

Reply:
This should serve for now. :)